src/app/pages/Languages.svelte

```diff
@@ -41,6 +41,21 @@
       },
     ],
   };
+  // TODO: Temporary data placeholder for autocorrect component and will be removed once #33 is done
+  let testingData: any = [
```

Comment:
The testing data has been removed from code in this pr [#152](https://github.com/eddieantonio/predictive-text-studio/pull/152), since I did it in a backward manner by building the component first which why I implement a placeholder testing data. I am sorry for the confusion😔, I will also rebase #125 once this pr goes in so to ease out the pr review process and confusion.